### PR TITLE
ELEC-479: Update messages

### PR DESCRIPTION
**Pull request by ckitagawa**
- Switches power consumption from motor power to total power.
- Use the aggregate voltage message.
- Use a poor approximation for SOC. Will add a better one in future (if time allows).

### CODE REVIEW

File: client/src/ts/dd_main.ts [hunk 1]
```diff
@@ -5,6 +5,21 @@ import vis = require('./visibility');
 import animate = require('./animate');
 import canDefs = require('./can_msg_defs');
 
+// SOC
+
+function mapSocPercent(voltage: number): number {
+  // Use a poor linear approximation for now based on:
+  // https://lygte-info.dk/info/BatteryChargePercent%20UK.html
+  // TODO(ckitagawa): Make this better if time allows.
+  const result = 3.15 * voltage - 373;
```
Comment (tituschow) [inline client/src/ts/dd_main.ts]: Magic formula? I don't see this anywhere in the page.

File: client/src/ts/dd_main.ts [hunk 2]
```diff
@@ -175,9 +185,10 @@ ws.onmessage = (event) => {
       // solarReadout.value(msg.data);
       break;
     case canDefs.CanMessage.CAN_MESSAGE_MOTOR_VELOCITY:
-      const value =
-          (msg.data.vehicle_velocity_left + msg.data.vehicle_velocity_right) /
-          2 * 0.036;  // cm/s->km/h
+      const value = Math.abs(
+          (((msg.data.vehicle_velocity_left << 32) >> 32) +
```
Comment (tituschow) [inline client/src/ts/dd_main.ts]: Should probably document this?

File: client/src/ts/dd_main.ts [hunk 3]
```diff
@@ -7,11 +7,14 @@ import canDefs = require('./can_msg_defs');
...
 // SOC
 
-function mapSocPercent(voltage: number): number {
-  // Use a poor linear approximation for now based on:
-  // https://lygte-info.dk/info/BatteryChargePercent%20UK.html
+function mapSocPercent(v: number): number {
+  // Based on a discharge curve fit of polynomial degree 6.
   // TODO(ckitagawa): Make this better if time allows.
-  const result = 3.15 * voltage - 373;
+  const max_capcity = 4368.3804;  // Ah
```
Comment (tituschow) [inline client/src/ts/dd_main.ts]: s/capcity/capacity/

4368Ah? Our pack should be ~126Ah.

Comment (ckitagawa) [reply client/src/ts/dd_main.ts]: All the curves stop at 122 but thanks for the catch. I accidentally multiplied by 36.

File: client/src/ts/dd_main.ts (hunk 1 above)
Comment (ckitagawa) [reply client/src/ts/dd_main.ts]: Added the sheet I used to confluence.

File: client/src/ts/dd_main.ts (hunk 2 above)
Comment (ckitagawa) [reply client/src/ts/dd_main.ts]: Done.

**Pull request merged**